packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -60,6 +60,8 @@ contract DeployImplementationsInput is BaseDeployIO {
     SuperchainConfig internal _superchainConfigProxy;
     ProtocolVersions internal _protocolVersionsProxy;
 
+    string internal _standardVersionsToml;
```

Comment:
Based on the above, this is supposed to be the content itself, and not a path to it right? What is the rationale for that?

Reply:
See my comment above.